financial-connections/src/main/java/com/stripe/android/financialconnections/model/FinancialConnectionsGenericInfoScreen.kt

```diff
@@ -0,0 +1,144 @@
+import android.os.Parcelable
+import com.stripe.android.financialconnections.model.Image
+import com.stripe.android.financialconnections.model.serializer.BodyEntrySerializer
+import kotlinx.parcelize.Parcelize
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Parcelize
+@Serializable
+internal data class FinancialConnectionsGenericInfoScreen(
+    val id: String,
+    val header: Header? = null,
+    val body: Body? = null,
+    val footer: Footer? = null,
+    val options: Options? = null
+) : Parcelable {
+
+    @Parcelize
+    @Serializable
+    internal data class Header(
+        val title: String? = null,
+        val subtitle: String? = null,
+        val icon: Image? = null,
+        val alignment: Alignment? = null
```

Comment:
Should values like this have defaults defined here?

Reply:
Even if they're nullable, if no default is added serialization will fail when they're not on a response. All these models are also nullable on frontend (figuring how to handle nullability will come on a follow-up PR)

Reply:
I mean if we should have a default alignment defined here as opposed to multiple places in the codebase. But if you want to handle nullability in a follow-up pull request, then I’ll wait for that.